stock/templates/registered/operationmanagement.html

```diff
@@ -23,57 +23,90 @@ <h1 class="page-header text-center">{% trans "Fleet Operation Timers" %}
-                <th class="text-center">{% trans "Local Time" %}</th>
-		<th class="text-center">{% trans "Duration" %}</th>
-                <th class="text-center">{% trans "FC" %}</th>
-                <th class="text-center">{% trans "Details" %}</th>
-		<th class="text-center">{% trans "Post Time" %}</th>
-		{% if perms.auth.optimer_management %}
-                	<th class="text-center">{% trans "Creator" %}</th>
-		        <th class="text-center">{% trans "Action" %}</th>
-                {% endif %}
-            </tr>
+     <h4><b>{% trans "Next Timers" %}</b></h4>
+     {% if future_timers %}
+     <table class="table">
+        <tr>
+            <th style="width:150px" class="text-center">{% trans "Operation Name" %}</th>
+            <th style="width:150px" class="text-center">{% trans "Doctrine" %}</th>
+            <th style="width:130px" class="text-center">{% trans "Form Up System" %}</th>
+            <th style="width:135px" class="text-center">{% trans "Start Time" %}</th>
+            <th style="width:130px" class="text-center">{% trans "Local Time" %}</th>
+            <th style="width:90px"  class="text-center">{% trans "Duration" %}</th>
+            <th style="width:105px" class="text-center">{% trans "FC" %}</th>
+            {% if perms.auth.optimer_management %}
+            <th class="text-center">{% trans "Creator" %}</th>
+            <th style="width:100px" class="text-center">{% trans "Action" %}</th>
+            {% endif %}
+        </tr>
+     {% for ops in future_timers %}
+        <tr>
+            <td class="text-center">{{ ops.operation_name }}</td>
+            <td class="text-center">{{ ops.doctrine }}</td>
+            <td class="text-center">
+                <a href="http://evemaps.dotlan.net/system/{{ ops.system }}">{{ ops.system }}</a>
+            </td>
+            <td class="text-center" nowrap>{{ ops.start | date:"Y-m-d H:i" }}</td>
+            <td class="text-center" nowrap><div id="localtime{{ ops.id }}"></div><div id="countdown{{ ops.id }}"></div></td>
+            <td class="text-center">{{ ops.duration }}</td>
+            <td class="text-center">{{ ops.fc }}</td>
+            {% if perms.auth.optimer_management %}
+                <td class="text-center">{{ ops.eve_character }}</td>
+                <td class="text-center">
+                    <a href="{% url 'auth_remove_optimer' ops.id %}" class="btn btn-danger">
+                    <span class="glyphicon glyphicon-remove"></span>
+                </a><a href="{% url 'auth_edit_optimer' ops.id %}" class="btn btn-info"><span class="glyphicon glyphicon-pencil"></span></a>
+                </td>
+            {% endif %}
+        </tr>
+    {% endfor %}
+    </table>
+    {% else %}
+        <div class="alert alert-warning text-center">{% trans "No upcoming timers." %}</div>
+    {% endif %}
+    
+     <h4><b>{% trans "Past Timers" %}</b></h4>
+     {% if past_timers %}
+     <table class="table">
```

Comment:
Given that this table is used twice verbatim (as far as I can see) it would be better to split it out into a sub template, and then include it using `{% "sub_template_name.html" with timers=future/past_timers %}` then rename your `past_timers` and `future_timers` variable to `timers` (provided its not used elsewhere).